frontend/src/scenes/organization/Settings/index.tsx

```diff
@@ -70,6 +74,38 @@ function EmailPreferences({ isRestricted }: RestrictedComponentProps): JSX.Eleme
     )
 }
 
+function PluginAlertPreference({ isRestricted }: RestrictedComponentProps): JSX.Element {
+    const { members, membersLoading } = useValues(membersLogic)
+    const { currentOrganization } = useValues(organizationLogic)
+    const { updateOrganization } = useActions(organizationLogic)
+
+    return (
+        <div>
+            <h2 id="plugin-notification-preferences" className="subtitle">
+                Sending plugin alerts
+            </h2>
+            <div>
+                Select members of your team who should receive alerts if a plugin gets disabled. If no-one is selected,
```

Comment:
we might have other kinds of alerts too, so I'd phrase plugin being disabled as an example

Reply:
Big +1, let's generalize this to system alerts.